### ISSUE TITLE
Knife

### ISSUE THREAD

**Issue by petravic** [Tommy228/TTT_Spectator_Deathmatch#4]
You can knife people in dm if they are infront of you!

**Comment by raymond4400** [comment]
Any updates on fixing this?

**Comment by markusmarkusz** [comment]
Not fixed at the moment.

**Comment by MinIsMin** [comment]
I tried and it didn't work. Which knife did you use? I used standard TTT knife.

**Comment by markusmarkusz** [comment]
Use the normal knife. It is possible.

**Comment by MinIsMin** [comment]
Will look into it.

**Comment by mcNuggets1** [comment]
We'll have to detour alot of shit.
It isn't worth it.

It's not just the knife, its every weapon using a traceline/tracehull etc.

**Comment by YetiYetYet** [comment]
I'm happy to see it's will will be fix

**Comment by PuNcAk3** [comment]
Currently having an issue with people in specdm, being able to be knifed. Not only that, but their accessories stay on, showing to the alive players. Seeing as we currently had 1.7, not sure if by 1.8.5 fixed this, but was just bringing it to your attention.

**Comment by MinIsMin** [comment]
Yep, that's the knife issue, I'm working on a solution but don't expect it anytime soon.
The accessoirs are an other issue, see #64

BTW: You should upgrade your SpecDM version as there were some exploit fixes:
https://github.com/Tommy228/TTT_Spectator_Deathmatch/releases
